.devcontainer/run_in_docker.sh

```
@@ -0,0 +1,169 @@
+  for comp_db in "$WORKSPACE_DIR"/.build-*/compile_commands.json "$WORKSPACE_DIR"/.build-*-container/compile_commands.json; do
+    if [ -f "$comp_db" ] && [ -s "$comp_db" ]; then
+      python3 -c '
+import sys
+with open(sys.argv[1], "r") as f:
+    content = f.read()
+content = content.replace(sys.argv[2], sys.argv[3])
+content = content.replace(sys.argv[4], sys.argv[5])
+with open(sys.argv[1], "w") as f:
+    f.write(content)
+' "$comp_db" "/workspaces/$WORKSPACE_BASENAME" "$WORKSPACE_DIR" "/opt/valkey-search-deps" "/tmp/valkey-search-deps" 2>/dev/null || true
+      rel_path="${comp_db#$WORKSPACE_DIR/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)